facade/fileFacadeFunctions.js

```diff
@@ -13,9 +13,9 @@ exports.uploadFacade = (file, issueId) => {
     
     return new Promise((resolve, reject) => {
 
-        saveModul.saveFile(file, issueId)
+        saveModul.saveFile(file)
             .then((params) => {
-                return resolve(DalsFileController.uploadFile(params));
```

Comment:
This is where you should be calling dals to save the Mongo file collection. Not in saveModul